keras_core/ops/image.py

```diff
@@ -244,3 +244,126 @@ def affine_transform(
+        interpolation: Interpolation method. Available methods are `"nearest"`,
+            and `"bilinear"`. Defaults to `"bilinear"`.
+        fill_mode: Points outside the boundaries of the input are filled
+            according to the given mode. Available methods are `"constant"`,
+            `"nearest"`, `"wrap"` and `"reflect"`. Defaults to `"constant"`.
+            Note that `"wrap"` is not supported by Torch backend.
+        fill_value: Value used for points outside the boundaries of the input if
+            `fill_mode="constant"`. Defaults to `0`.
+        data_format: string, either `"channels_last"` or `"channels_first"`.
+            The ordering of the dimensions in the inputs. `"channels_last"`
+            corresponds to inputs with shape `(batch, height, width, channels)`
+            while `"channels_first"` corresponds to inputs with shape
+            `(batch, channels, height, weight)`. It defaults to the
+            `image_data_format` value found in your Keras config file at
+            `~/.keras/keras.json`. If you never set it, then it will be
+            `"channels_last"`.
+
+    Returns:
+        Applied affine transform image or batch of images.
+
+    Examples:
+
+    >>> x = np.random.random((2, 64, 80, 3)) # batch of 2 RGB images
+    >>> transform = np.array(
+    ...     [
+    ...         [1.5, 0, -20, 0, 1.5, -16, 0, 0],  # zoom
+    ...         [1, 0, -20, 0, 1, -16, 0, 0],  # translation
+    ...     ]
+    ... )
+    >>> y = keras_core.ops.image.affine_transform(x, transform)
+    >>> y.shape
+    (2, 64, 80, 3)
+
+    >>> x = np.random.random((64, 80, 3)) # single RGB image
+    >>> transform = np.array([1.0, 0.5, -20, 0.5, 1.0, -16, 0, 0])  # shear
+    >>> y = keras_core.ops.image.affine_transform(x, transform)
+    >>> y.shape
+    (64, 80, 3)
+
+    >>> x = np.random.random((2, 3, 64, 80)) # batch of 2 RGB images
+    >>> transform = np.array(
+    ...     [
+    ...         [1.5, 0, -20, 0, 1.5, -16, 0, 0],  # zoom
+    ...         [1, 0, -20, 0, 1, -16, 0, 0],  # translation
+    ...     ]
+    ... )
+    >>> y = keras_core.ops.image.affine_transform(x, transform,
+    ...     data_format="channels_first")
+    >>> y.shape
+    (2, 3, 64, 80)
+    """
+    # if any_symbolic_tensors((image)):
+    #     return ExtractPatches(
+    #         sizes, strides, rates, padding, data_format=data_format,
+    #     ).symbolic_call(image)
+    patch_h, patch_w = sizes[0], sizes[1] 
+    if data_format == 'channels_last':
+        channels_in = image.shape[-1]
+        out_dim = image.shape[1] // patch_h * image.shape[2] // patch_w
+    elif data_format == 'channels_first':
+        # if batched
+        channels_in = image.shape[1]
+        out_dim = (image.shape[2] // patch_h) * (image.shape[3] // patch_w)
+    kernel = ops.eye(patch_h * patch_w * channels_in, out_dim)
+    kernel = ops.reshape(kernel, (patch_h, patch_w, channels_in, out_dim))
+    return ops.nn.conv(image, kernel, 14, 'VALID')
```

Comment:
- Padding should be lowercase
- Prefer using keyword arguments for better readbility